addons/postgresql/templates/componentdefinition-15.yaml

```diff
@@ -201,6 +196,10 @@ spec:
           runAsUser: 0
         command:
           - /kb-scripts/setup.sh
+        lifecycle:
+          preStop:
+            exec: 
+              command: ["/kb-scripts/etcd-clean.sh"]
```

Comment:
It seems not appropriate for this to be executed when each Pod is terminating

Reply:
The first Pod in $KB_CLUSTER_COMPONENT_POD_IP_LIST is selected and its terminate API is called

Reply:
And all keys 're deleted